### ISSUE TITLE
1.4.1 Use of Colour - G14: Ensuring that information conveyed by color differences is also available in text

### ISSUE THREAD

**Issue by josephlabrecque** [dulibrarytech/digitaldu-frontend#148]

**Comment by josephlabrecque** [comment]
I cannot locate any instances where the application refers to color as an identifier at all - so this one should be good.
